### PR TITLE
change .ember-cli to .ember-cli.js

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
now we don't need package.json shenanigans for MU

### CODE REVIEW

File: blueprints/module-unification-app/files/.ember-cli.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-{
+process.env.EMBER_CLI_MODULE_UNIFICATION = true;
```
Comment (rwjblue) [inline blueprints/module-unification-app/files/.ember-cli.js]: Lets add `'use strict';` up here too....

**Comment by ppcano** [comment]
@NullVoxPopuli This change could also be needed in the the MU addon blueprint.

https://github.com/ember-cli/ember-cli/blob/master/blueprints/module-unification-addon/files/.ember-cli

**Comment by NullVoxPopuli** [comment]
@ppcano @rwjblue rebased and copied the .ember-cli.js to mu-addon.

questions re: octane 
- will there be a separate ocntane addon blueprint repo like there is for the octane app blueprint?
- should all the other .ember-cli files change over to .js files?

**Comment by NullVoxPopuli** [comment]
Unsure. Afaik, remote blueprint packages have a rigid structure, where there is a single files directory at the repo root. I haven't looked at the blueprint resolution code yet, though... So.. idk if it'd be possible. It'd be fantastic if it were though. 
I'd also like to use variables and symlinks for a bunch of the duplicate stuff

**Comment by Turbo87** [comment]
FWIW I'm not a fan of this change. The file currently can be statically parsed by e.g. IDE, which will no longer be the case once we allow JS. IMHO we need stronger reasons to change this than just making a temporary workaround work a little nicer.

**Comment by rwjblue** [comment]
> The file currently can be statically parsed by e.g. IDE, which will no longer be the case once we allow JS

FWIW, we already allow `.ember-cli.js`.

**Comment by locks** [comment]
Long live MU! Thanks for the contributions.

**Pull request closed without merging**